src/PresentationalComponents/ConditionalLink/ConditionalLink.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+/**
+ * This is a dumb component that only recieves properties from a smart component.
```

Comment:
We should start trying to get rid of these "dumb component" comments. We should try to document our components a little bit to help readability.